models/balancer/ethereum/balancer_ethereum_balances.sql

```diff
@@ -0,0 +1,71 @@
+{{
+    config(
+        alias='balancer_balances',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "balancer",
+                                    \'["masquot", "jacektrocinski"]\') }}'
+    ) 
+}}
+
+WITH pools AS (
+    SELECT pool as pools
+    FROM {{ source('balancer_ethereum', 'BFactory_evt_LOG_NEW_POOL') }}
+),
+
+joins AS (
+    SELECT p.pools as pool, date_trunc('day', e.evt_block_time) AS day, e.contract_address AS token, SUM(value) AS amount
+    FROM {{ source('erc20_ethereum', 'evt_transfer') }} e
+    INNER JOIN pools p ON e.`to` = p.pools
+    GROUP BY 1, 2, 3
+    UNION ALL
+    SELECT e.`to` as pool, date_trunc('day', e.evt_block_time) AS day, e.contract_address AS token, SUM(value) AS amount
+    FROM {{ source('erc20_ethereum', 'evt_transfer') }} e
+    WHERE e.`to` = '0xba12222222228d8ba445958a75a0704d566bf2c8'
```

Comment:
for hardcoded addresses tied to a contract used in filters, a good approach we're looking to adopt is a variable at the top of the model, followed by it's reference used throughout the script.
example:
https://github.com/duneanalytics/spellbook/blob/main/models/curvefi/ethereum/curvefi_ethereum_trades.sql#L16
https://github.com/duneanalytics/spellbook/blob/main/models/curvefi/ethereum/curvefi_ethereum_trades.sql#L789

Reply:
applies to line 36 as well

Reply:
Fixed.

Added the Jinja variable `balancer_contract`.